dashboard_website/index.js

```diff
@@ -0,0 +1,419 @@
+        if (newValue >= 2) {
+            rangeSliderLower.value = newValue;
+            updateRangeLower(newValue);
+        }
+    });
+
+    incrementRangeLower.addEventListener('click', function() {
+        let newValue = parseInt(rangeSliderLower.value) + 2;
+        if (newValue <= rangeSliderLower.max) {
+            rangeSliderLower.value = newValue;
+            updateRangeLower(newValue);
+        }
+    });
+
+    // Add event listener to region select
+    const regionSelect = document.getElementById('region-select');
+    regionSelect.addEventListener('change', function(event) {
+        selectedRegion = event.target.value;
+        updateRangeRegion('10')
+        updateRangeLowerRegion('2')
+        drawRegionBoxPlot();
+    });
+
+    // Add event listener to SNR select
+    const snrRegionSelect = document.getElementById('snr-region-select');
+    snrRegionSelect.addEventListener('change', function(event) {
+        selectedSNRRegion = event.target.value;
+        updateRangeRegion('10')
+        updateRangeLowerRegion('2')
+        drawRegionBoxPlot();
+    });
+
+    // Add event listener to type select
+    const typeRegionSelect = document.getElementById('type-region-select');
+    typeRegionSelect.addEventListener('change', function(event) {
+        selectedTypeRegion = event.target.value;
+        updateRangeRegion('10')
+        updateRangeLowerRegion('2')
+        drawRegionBoxPlot();
+    });
+
+    // Add event listeners to range region slider and buttons
+    const rangeSliderRegion = document.getElementById('range-slider-region');
+    const rangeValueRegion = document.getElementById('range-value-region');
+    const decrementRangeRegion = document.getElementById('decrement-range-region');
+    const incrementRangeRegion = document.getElementById('increment-range-region');
+
+    function updateRangeRegion(value) {
+        selectedRangeRegion = value;
+        //rangeValueRegion.textContent = value;
+        rangeSliderRegion.value = value
+        drawRegionBoxPlot();
+    }
+
+    rangeSliderRegion.addEventListener('input', function(event) {
+        updateRangeRegion(event.target.value);
+    });
+
+    decrementRangeRegion.addEventListener('click', function() {
+        let newValue = parseInt(rangeSliderRegion.value) - 2;
+        if (newValue >= 2) {
+            rangeSliderRegion.value = newValue;
+            updateRangeRegion(newValue);
+        }
+    });
+
+    incrementRangeRegion.addEventListener('click', function() {
+        let newValue = parseInt(rangeSliderRegion.value) + 2;
+        if (newValue <= rangeSliderRegion.max) {
+            rangeSliderRegion.value = newValue;
+            updateRangeRegion(newValue);
+        }
+    });
+
+    // Add event listeners to range slider and buttons for region
+    const rangeSliderLowerRegion = document.getElementById('lower-range-slider-region');
+    const rangeValueLowerRegion = document.getElementById('lower-range-value-region');
```

Comment:
This seems to be inverted from what I would expect. Clicking `+` on the lower slider decreases the minimum rather than increasing it.

Also, I noticed a case where the max and min were the same value so clicking it didn't do anything. Maybe it could be disabled in those cases?

Reply:
> This seems to be inverted from what I would expect. Clicking + on the lower slider decreases the minimum rather than increasing it.

Fixed. I thought it made sense this way because clicking `+` expands the view for more data points.

> Also, I noticed a case where the max and min were the same value so clicking it didn't do anything. Maybe it could be disabled in those cases?

Done.